### DISCUSSION TITLE
Account db design re-implementation

### DISCUSSION THREAD

**Discussion by einhverfr** [ledgersmb/LedgerSMB#6966] [Design]
See https://github.com/ledgersmb/LedgerSMB/wiki/Chart-of-Accounts-Framework-Database-Design

Need db schema tested, fleshed out, and stored process written/created.

**Comment by einhverfr** [comment]
db schema tests required:

* cannot share ids between headings and accounts (exception thrown)
* cannot have a cycle in headings.

**Reply by ehuelsmann** [reply]
The second bullet (no cycle) has been long implemented.

**Comment by ehuelsmann** [comment]
Shared Id values pose a problem and presumably will always be a problem due to the fact that inherited tables do not prevent ID overlap.
